app/views/listings/form/_currency_selector.haml

```diff
@@ -1,3 +1,2 @@
 .currency-selector
   = Money.new(1, @current_community.currency).symbol
-  = form.hidden_field :currency, :value => @current_community.currency
```

Comment:
@lyyder @kpuputti I added one more commit that reintroduced currency handling in controller. https://github.com/sharetribe/sharetribe/pull/2842/commits/b533b65174a3028dbf81bae404823c4937bb152b